src/main.cpp

```diff
@@ -2986,6 +2986,8 @@ bool CVerifyDB::VerifyDB(CCoinsView *coinsview, int nCheckLevel, int nCheckDepth
             } else
                 nGoodTransactions += block.vtx.size();
         }
+        if (ShutdownRequested())
+            return true;
```

Comment:
Hm okay I guess that makes sense, that's what you get with two possible return codes, and extending it isn't worth the extra trouble either...